packages/logger/tests/unit/middleware/middy.test.ts

```diff
@@ -153,6 +153,64 @@ describe('Middy middleware', () => {
       });
 
     });
+
+  });
+
+  describe('Feature: clear state', () => {
+
+    test('when enabled, the persistent log attributes added in the handler are removed after the handler\'s code is executed', async () => {
```

Comment:
(Questions) should they be removed?  From the code line 195-206, they are not removed but persisted through out handler executions.